cache/models/samples.js

```diff
@@ -1010,7 +801,7 @@ module.exports = {
    * @returns {Promise} - Resolves to a list of all samples objects
    */
   findSamples(req, res, logObject) {
-    const opts = getOptionsFromReq(req.swagger.params);
+    const opts = modelUtils.getOptionsFromReq(req.swagger.params, helper);
```

Comment:
need the helper as a param, as helper can be subject or sample or another resource's helper.